dali/pipeline/operators/reader/parser/CMakeLists.txt

```diff
@@ -29,7 +35,12 @@ SET_TARGET_PROPERTIES(CAFFE2_PROTO PROPERTIES COMPILE_FLAGS "-include dali/api_h
 
 # TF
 if (BUILD_PROTO3)
-  protobuf_generate_cpp(TF_PROTO_SRCS TF_PROTO_HEADERS EXPORT_MACRO DLL_PUBLIC proto/example.proto proto/feature.proto)
+  if(${CMAKE_VERSION} VERSION_LESS "3.9.0")
```

Comment:
This way you expose all protobuf symbols we want to hide. I will address this by providing custom protobuf_generate_cpp shipped with Dali.

Reply:
so is there any thing I need to do?

Reply:
If the target is to hide all symbols in ```*.pb.cc```, the universal way is adding a static library for every ```.proto``` file, and DALI will deal with these library in the way dealing with CUDA static library. 
Or only the implementation of the protobuf message is expected to be hidden?